packages/web-components/fast-components/src/radio-group/radio-group.template.ts

```diff
@@ -0,0 +1,14 @@
+import { html, slotted } from "@microsoft/fast-element";
+import { RadioGroup } from "./radio-group";
+
+export const RadioGroupTemplate = html<RadioGroup>`
+    <template
+        role="radiogroup"
+        ?aria-disabled="${x => x.disabled}"
+        ?aria-readonly="${x => x.readOnly}"
+    >
+        <div part="control" class="control">
```

Comment:
Do we need this div? It seems like the `:host` can serve this purpose based on the current markup and model.

Reply:
removed